src/pulselistener/pulselistener/lib/pulse.py

```diff
@@ -109,3 +101,50 @@ def run_consumer(consumer):
             pass
         event_loop.close()
         sys.exit()
+
+
+class PulseListener(object):
+    '''
+    Pulse queues connector to receive external messages and react to them
+    '''
+    QUEUE_OUT = 'pulse:out'
```

Comment:
Similarly for WebServer, it could be made generic and each client could pass the name of the queue. As you said this would allow one client to listen to more than one pulse message.

We could also do this later on though, if we find it necessary.